src/Microsoft.AspNetCore.Server.IntegrationTesting/xunit/SkipIfCurrentRuntimeIsClrAttribute.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using Microsoft.AspNetCore.Testing.xunit;
+
+namespace Microsoft.AspNetCore.Server.IntegrationTesting
+{
+    /// <summary>
+    /// Skips a test if the runtime used to run the test is Clr.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = false)]
+    public class SkipIfCurrentRuntimeIsClrAttribute : Attribute, ITestCondition
+    {
+        public bool IsMet
+        {
+            get
+            {
+                return !Process.GetCurrentProcess().ProcessName.ToLower().Contains("testhost");
```

Comment:
I was going to use it in IISIntegration functional tests. Also I don't see why we only have the attribute for skipping CoreCLR but not CLR.